### PR TITLE
Feature/copy update policies

### LINKED ISSUE #56: Policies to be included in import, copy commands
command "copy" and "update" don't copy/update policies

### PR DESCRIPTION
**Pull request by eguzki**
`3scale copy service` and `3scale update service` commands add proxy policy to the list of entities to copy/update.

Needs new release of 3scale-api-ruby with the following PR merged
3scale/3scale-api-ruby#49

Fixes #56

### CODE REVIEW

**Comment by eguzki** [comment]
ready for review @mikz  @miguelsorianod

File: lib/3scale_toolbox/commands/copy_command/copy_service.rb [hunk 1]
```diff
@@ -40,7 +40,8 @@ def run
             Tasks::CopyApplicationPlansTask.new(context),
             Tasks::CopyLimitsTask.new(context),
             Tasks::DestroyMappingRulesTask.new(context),
-            Tasks::CopyMappingRulesTask.new(context)
+            Tasks::CopyMappingRulesTask.new(context),
+            Tasks::CopyPoliciesTask.new(context)
```
Comment (mikz) [inline lib/3scale_toolbox/commands/copy_command/copy_service.rb]: better to have a trailing comma there IMO, to reduce unnecessary changes when adding items

Comment (eguzki) [reply lib/3scale_toolbox/commands/copy_command/copy_service.rb]: :+1:
